worlds/stardew_valley/test/TestMods.py

```diff
@@ -174,19 +174,3 @@ def test_mod_entrance_randomization(self):
 
                 self.assertEqual(len(set(randomized_connections.values())), len(randomized_connections.values()),
                                  f"Connections are duplicated in randomization. Seed = {seed}")
-
-
-class TestModTraps(SVTestBase):
```

Comment:
Why did you get rid of this one?

Reply:
Because there are no traps related to magic anymore.